pkgs/by-name/ma/matrix-synapse-unwrapped/package.nix

```diff
@@ -31,7 +31,8 @@ python3Packages.buildPythonApplication rec {
 
   postPatch = ''
     substituteInPlace pyproject.toml \
-      --replace-fail "setuptools_rust>=1.3,<=1.11.1" "setuptools_rust<=1.12,>=1.3"
+      --replace-fail "setuptools_rust>=1.3,<=1.11.1" "setuptools_rust<=1.12,>=1.3" \
```

Comment:
```suggestion
      --replace-fail "setuptools_rust>=1.3,<=1.11.1" setuptools_rust \
```

Reply:
This is a build time dependency. If matrix-synapse fails to build, we'll surely notice. The upper bound just makes it more likely to fail to build for no reason.

Reply:
The reason I bring this up is, I fixed this before and that's when I stumbled upon https://github.com/element-hq/synapse/blob/develop/pyproject.toml#L369-L376 (I'd like to point out the "runtime errors" part).

That's at least why I didn't change it to your suggestion back then.